core/src/main/java/org/openstack4j/model/compute/ServerAction.java

```diff
@@ -0,0 +1,61 @@
+package org.openstack4j.model.compute;
+
+import java.util.Date;
+import java.util.List;
+
+import org.openstack4j.model.ModelEntity;
+import org.openstack4j.model.common.ServerActionEvent;
+
+/**
+ * This interface describes the getter-methods (and thus components) of a Event.
```

Comment:
It is `ServerAction` now. Better to refer to symbols using `{@link ...}` to catch these errors.